base/kra/src/main/java/com/netscape/kra/TokenKeyRecoveryService.java

```diff
@@ -349,7 +406,6 @@ public synchronized boolean serviceRequest(IRequest request) throws EBaseExcepti
             CryptoToken internalToken =
             CryptoManager.getInstance().getInternalKeyStorageToken();
             */
-            CryptoToken token = mStorageUnit.getToken();
```

Comment:
you took out this line, and then put it back 200 lines before.  But token isn't used till the line right after this line.